README.md

```diff
@@ -191,6 +192,36 @@ You can edit prefix & group route name as you want, default this plugin use mode
     }
 ```
 
+### Middlewares
+
+You can add or override middlewares at two specific places. Via the Filament Panel Provider and/or via the Resources $routeMiddleware.
+
+If you set `route.use_resource_middlewares` to true, the package will register the middlewares for that specific resource as defined in:
+
+```php
+class BlogResource extends Resource
+    {
+        ...
+        protected static string | array $routeMiddleware = []; // <-- your specific resource middlewares
```

Comment:
please create interface called `HasRouteMiddleware` and there are property $routeMiddleware and getter method `getRouteMiddleware`

Reply:
`protected static string | array $routeMiddleware = [];` is something from filament resource itself.

check filament/filament/src/Resources/Resource.php

Reply:
ah i see, i never use it before.